### ISSUE TITLE
Action only replaces the first occurence, not all

### ISSUE THREAD

**Issue by MrVauxs** [richardrigutins/replace-in-files#134]
I have a script that automatically modifies some JSON information on repo creation (`on: create:`). I have come across an issue however that the action does not replace all instances of the `example-id` with the name of the repo `${{ github.event.repository.name }}` in the JSON file.

Here is an abbreviated version of what I am working with and the result of the action:
## Before
```json
{
	"id": "module-template",
	"title": "Example Title",
	"packs": [
		{
			"label": "Items - Example Title"
		}
	],
	"manifest": "https://example.url/module-template-randomhash/module.json"
}
```

## After
```json
{
	"id": "specific-id",
	"title": "Specific Title",
	"packs": [
		{
			"label": "Items - Example Title"
		}
	],
	"manifest": "https://example.url/module-template-randomhash/module.json"
}
```

**Comment by MrVauxs** [comment]
The action uses `.replace(a, b)` instead of `.replaceAll(a, b)`

https://github.com/richardrigutins/replace-in-files/blob/386893d8dd77a9d88a80ff2c4d4d7ba0a9bbff03/src/utils.ts#L97

**Comment by richardrigutins** [comment]
Thank you @MrVauxs!
The issue should now be fixed in the latest v2 release (v2.1.2).
